src/OpenIddict.Core/OpenIddictBuilder.cs

```diff
@@ -277,6 +279,59 @@ public class OpenIddictBuilder {
         }
 
         /// <summary>
+        /// Registers a new ephemeral key used to sign the tokens issued by OpenIddict: the key
+        /// is discarded when the application shuts down and tokens signed using this key are
+        /// automatically invalidated. This method should only be used during development:
+        /// on production, using a X.509 certificate stored in the machine store is recommended.
+        /// </summary>
+        /// <returns>The <see cref="OpenIddictBuilder"/>.</returns>
+        public virtual OpenIddictBuilder AddEphemeralSigningKey() {
```

Comment:
FYI, this helper is identical to ASOS' `AddEphemeralKey()` helper but we can't use it as it's not part of beta6. We'll migrate to the "official" helper when moving to beta7.